armi/settings/settingsRules.py

```diff
@@ -12,7 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""This module contains rules supporting custom actions taken on the reading of setting inputs"""
+"""
+This module contains rules supporting custom actions taken on the reading of setting inputs
+
+Note
+----
+The ``settingsRules`` system was developed before the pluginification of ARMI, and
+doesn't play very nicely with the post-plugin world. This is mainly because
+registration of new rules happens at import time of the rule itself, which is
+unreliable and difficult to control, and even sort of implicit and sneaky. They are
+also somewhate redundant with the
+:py:class:`armi.operators.settingsValidation.Query`/:py:class:`armi.operators.settingsValidation.Inspector`
```

Comment:
might need spaces by `/` for sphinx to handle, not sure.

Reply:
Seems happy with it